lib/std/sysrand.nim

```diff
@@ -91,13 +91,16 @@ when defined(js):
     const batchSize = 256
 
     proc getRandomValues(p: Uint8Array) {.importjs: "window.crypto.getRandomValues(#)".}
+      # The requested length of `p` must not be more than 65536.
```

Comment:
```suggestion
      doAssert p.len <= 65536
```
better check than assume

Reply:
I mean `getRandomValues` has this restriction. But I use a batch impl internal, so it doesn't hurt.

Reply:
makes sense. should we have `const batchSize = 256` or `const batchSize = 65536` for js?

Reply:
I'm not sure.